### ISSUE TITLE
[Bug]: Auto Delete Stopped Working After Upgrade to Version 0.4.4

### ISSUE THREAD

**Issue by birdwing** [tubearchivist/tubearchivist#622]
### I've read the documentation

- [X] I'm running the latest version of Tube Archivist and have read the [release notes](https://github.com/tubearchivist/tubearchivist/releases/latest).
- [X] I have read the [how to open an issue](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#how-to-open-an-issue) guide, particularly the [bug report](https://github.com/tubearchivist/tubearchivist/blob/master/CONTRIBUTING.md#bug-report) section.

### Operating System

Linux

## Describe the bug
Videos are not being deleted automatically.

## Steps To Reproduce
Go to Settings -> Application
Set Auto Delete Watched Videos to **1**
Download a Video to Trigger Auto Delete
Old videos are not deleted

## Expected behavior
A day after the videos are watched they should be deleted.

### Relevant log output

```shell
{"log":"[2023-12-03 08:47:59,821: INFO/MainProcess] Task download_pending[156a59a2-1671-4a06-98c3-098cae2cc08a] received\n","stream":"stderr","time":"2023-12-03T13:47:59.821750745Z"}
{"log":"[2023-12-03 08:47:59,825: WARNING/ForkPoolWorker-4] download_pending create callback\n","stream":"stderr","time":"2023-12-03T13:47:59.826418618Z"}
{"log":"[2023-12-03 08:48:00,900: WARNING/ForkPoolWorker-4] auto delete older than 1 days\n","stream":"stderr","time":"2023-12-03T13:48:00.901252608Z"}
{"log":"[2023-12-03 08:48:00,937: WARNING/ForkPoolWorker-4] UC4O9HKe9Jt5yAhKuNv3LXpQ: delete older than 1d\n","stream":"stderr","time":"2023-12-03T13:48:00.937427682Z"}
{"log":"[2023-12-03 08:48:00,968: WARNING/ForkPoolWorker-4] UCWFKCr40YwOZQx8FHU_ZqqQ: delete older than 1d\n","stream":"stderr","time":"2023-12-03T13:48:00.968927068Z"}
{"log":"[2023-12-03 08:48:01,006: WARNING/ForkPoolWorker-4] 156a59a2-1671-4a06-98c3-098cae2cc08a success callback\n","stream":"stderr","time":"2023-12-03T13:48:01.006996146Z"}
{"log":"[2023-12-03 08:48:01,011: INFO/ForkPoolWorker-4] Task download_pending[156a59a2-1671-4a06-98c3-098cae2cc08a] succeeded in 1.1817593285813928s: None\n","stream":"stderr","time":"2023-12-03T13:48:01.011763178Z"}
{"log":"[2023-12-03 08:48:01,011: WARNING/ForkPoolWorker-4] 156a59a2-1671-4a06-98c3-098cae2cc08a return callback\n","stream":"stderr","time":"2023-12-03T13:48:01.011908956Z"}
{"log":"{\"_index\":\"ta_video\",\"_id\":\"YWp6qjevHH0\",\"found\":false}\n","stream":"stderr","time":"2023-12-03T13:49:37.525230258Z"}
{"log":"Not Found: /api/video/YWp6qjevHH0/\n","stream":"stderr","time":"2023-12-03T13:49:37.526428298Z"}
{"log":"[pid: 38|app: 0|req: 90/90] 10.0.0.41 () {58 vars in 1003 bytes} [Sun Dec  3 08:49:37 2023] GET /api/video/YWp6qjevHH0/ =\u003e generated 493 bytes in 20 msecs (HTTP/1.1 404) 9 headers in 353 bytes (1 switches on core 0)\n","stream":"stderr","time":"2023-12-03T13:49:37.527313105Z"}
{"log":"[2023-12-03 08:49:38,114: INFO/MainProcess] Task extract_download[874554fb-09b4-4ba8-81a7-e85431c54a27] received\n","stream":"stderr","time":"2023-12-03T13:49:38.11465222Z"}
{"log":"[2023-12-03 08:49:38,116: WARNING/ForkPoolWorker-4] extract_download create callback\n","stream":"stderr","time":"2023-12-03T13:49:38.116846028Z"}
{"log":"[2023-12-03 08:49:38,575: WARNING/ForkPoolWorker-4] YWp6qjevHH0: [1/1]: add to queue\n","stream":"stderr","time":"2023-12-03T13:49:38.5761514Z"}
{"log":"[2023-12-03 08:49:40,763: WARNING/ForkPoolWorker-4] 874554fb-09b4-4ba8-81a7-e85431c54a27 success callback\n","stream":"stderr","time":"2023-12-03T13:49:40.764372647Z"}
{"log":"[2023-12-03 08:49:40,771: INFO/ForkPoolWorker-4] Task extract_download[874554fb-09b4-4ba8-81a7-e85431c54a27] succeeded in 2.647122242487967s: None\n","stream":"stderr","time":"2023-12-03T13:49:40.771560117Z"}
{"log":"[2023-12-03 08:49:40,771: WARNING/ForkPoolWorker-4] 874554fb-09b4-4ba8-81a7-e85431c54a27 return callback\n","stream":"stderr","time":"2023-12-03T13:49:40.771738575Z"}
{"log":"[2023-12-03 08:49:51,536: INFO/MainProcess] Task download_pending[d674893c-a3cd-4027-89b4-82c642c299ba] received\n","stream":"stderr","time":"2023-12-03T13:49:51.536638631Z"}
{"log":"[2023-12-03 08:49:51,537: WARNING/ForkPoolWorker-4] download_pending create callback\n","stream":"stderr","time":"2023-12-03T13:49:51.538420901Z"}
{"log":"[2023-12-03 08:49:51,901: WARNING/ForkPoolWorker-4] YWp6qjevHH0: Downloading video\n","stream":"stderr","time":"2023-12-03T13:49:51.901434743Z"}
{"log":"[2023-12-03 08:50:01,572: WARNING/ForkPoolWorker-4] YWp6qjevHH0: get metadata from youtube\n","stream":"stderr","time":"2023-12-03T13:50:01.572988444Z"}
{"log":"[2023-12-03 08:50:03,355: WARNING/ForkPoolWorker-4] UCdBK94H6oZT2Q7l0-b0xmMg: get metadata from es\n","stream":"stderr","time":"2023-12-03T13:50:03.355392084Z"}
{"log":"[2023-12-03 08:50:03,363: WARNING/ForkPoolWorker-4] {\"_index\":\"ta_channel\",\"_id\":\"UCdBK94H6oZT2Q7l0-b0xmMg\",\"found\":false}\n","stream":"stderr","time":"2023-12-03T13:50:03.363715721Z"}
{"log":"[2023-12-03 08:50:03,363: WARNING/ForkPoolWorker-4] UCdBK94H6oZT2Q7l0-b0xmMg: get metadata from youtube\n","stream":"stderr","time":"2023-12-03T13:50:03.364151119Z"}
{"log":"[2023-12-03 08:50:04,174: WARNING/ForkPoolWorker-4] UCdBK94H6oZT2Q7l0-b0xmMg: download channel thumbnail\n","stream":"stderr","time":"2023-12-03T13:50:04.175345473Z"}
{"log":"[2023-12-03 08:50:05,273: WARNING/ForkPoolWorker-4] YWp6qjevHH0: get sponsorblock timestamps\n","stream":"stderr","time":"2023-12-03T13:50:05.273764259Z"}
{"log":"[2023-12-03 08:50:06,466: WARNING/ForkPoolWorker-4] auto delete older than 1 days\n","stream":"stderr","time":"2023-12-03T13:50:06.470616314Z"}
{"log":"[2023-12-03 08:50:06,505: WARNING/ForkPoolWorker-4] UC4O9HKe9Jt5yAhKuNv3LXpQ: delete older than 1d\n","stream":"stderr","time":"2023-12-03T13:50:06.505571602Z"}
{"log":"[2023-12-03 08:50:06,540: WARNING/ForkPoolWorker-4] UCWFKCr40YwOZQx8FHU_ZqqQ: delete older than 1d\n","stream":"stderr","time":"2023-12-03T13:50:06.541346029Z"}
{"log":"[2023-12-03 08:50:06,628: WARNING/ForkPoolWorker-4] d674893c-a3cd-4027-89b4-82c642c299ba success callback\n","stream":"stderr","time":"2023-12-03T13:50:06.628946087Z"}
{"log":"[2023-12-03 08:50:06,634: INFO/ForkPoolWorker-4] Task download_pending[d674893c-a3cd-4027-89b4-82c642c299ba] succeeded in 15.090998463332653s: 'downloaded 1 videos.'\n","stream":"stderr","time":"2023-12-03T13:50:06.635253769Z"}
{"log":"[2023-12-03 08:50:06,635: WARNING/ForkPoolWorker-4] d674893c-a3cd-4027-89b4-82c642c299ba return callback\n","stream":"stderr","time":"2023-12-03T13:50:06.635497112Z"}
{"log":"[2023-12-03 08:50:06,645: WARNING/ForkPoolWorker-4] A Connection error occurred sending JSON notification to 10.0.0.142.\n","stream":"stderr","time":"2023-12-03T13:50:06.645493193Z"}
```

### Anything else?

It doesn't even appear to attempt to delete the videos.
This is all I got in the logs after a fresh restart and setting DJANGO_DEBUG = true.
I attempted 2 downloads. Some of the videos not being deleted were watched 3 or 4 days ago.

Digging further into, I went to one of the Channel pages then to About.
I noticed that "Auto Delete" on each channel page has been set to "false"
I never did this. I have only ever set Auto Delete on the settings page.
It appears that somehow during the upgrade to this version, all the channel page setting overrides for Auto Delete got set to "false" (0)

**Comment by Pete060490** [comment]
Same here, just pasting my logs in case that helps:

```
[2023-12-13 19:32:50,625: INFO/MainProcess] Task extract_download[926d7100-5ddf-4dae-8be3-59261943fed2] received
[2023-12-13 19:32:50,627: WARNING/ForkPoolWorker-8] extract_download create callback
[2023-12-13 19:32:50,816: WARNING/ForkPoolWorker-8] vM6WtOmy190: [1/1]: add to queue
[2023-12-13 19:32:53,373: INFO/MainProcess] Task download_pending[529ad129-17de-40be-bc8f-69ca227c66b9] received
[2023-12-13 19:32:53,374: WARNING/ForkPoolWorker-1] download_pending create callback
[2023-12-13 19:32:53,375: WARNING/ForkPoolWorker-8] 926d7100-5ddf-4dae-8be3-59261943fed2 success callback
[2023-12-13 19:32:53,379: INFO/ForkPoolWorker-8] Task extract_download[926d7100-5ddf-4dae-8be3-59261943fed2] succeeded in 2.748674105037935s: None
[2023-12-13 19:32:53,379: WARNING/ForkPoolWorker-8] 926d7100-5ddf-4dae-8be3-59261943fed2 return callback
[2023-12-13 19:32:53,511: WARNING/ForkPoolWorker-1] vM6WtOmy190: Downloading video
[2023-12-13 19:33:04,489: WARNING/ForkPoolWorker-1] vM6WtOmy190: get metadata from youtube
[2023-12-13 19:33:06,851: WARNING/ForkPoolWorker-1] UCiW7-IEQCTqS9l7223whHZA: get metadata from es
[2023-12-13 19:33:06,977: WARNING/ForkPoolWorker-1] vM6WtOmy190: get ryd stats
[2023-12-13 19:33:07,086: WARNING/ForkPoolWorker-1] vM6WtOmy190: get sponsorblock timestamps
[2023-12-13 19:33:07,285: WARNING/ForkPoolWorker-1] vM6WtOmy190: sponsorblock failed: 404
[2023-12-13 19:33:07,287: WARNING/ForkPoolWorker-1] vM6WtOmy190-en: get user uploaded subtitles
[2023-12-13 19:33:07,287: WARNING/ForkPoolWorker-1] vM6WtOmy190-en: get auto generated subtitles
[2023-12-13 19:33:07,287: WARNING/ForkPoolWorker-1] vM6WtOmy190-de: get user uploaded subtitles
[2023-12-13 19:33:07,287: WARNING/ForkPoolWorker-1] vM6WtOmy190-de: get auto generated subtitles
[2023-12-13 19:33:08,146: WARNING/ForkPoolWorker-1] auto delete older than 1 days
[2023-12-13 19:33:08,206: WARNING/ForkPoolWorker-1] vM6WtOmy190: get comments
[2023-12-13 19:33:12,252: WARNING/ForkPoolWorker-1] WARNING: [youtube] Incomplete data received. Retrying (1/3)...
[2023-12-13 19:33:12,339: WARNING/ForkPoolWorker-1] WARNING: [youtube] Incomplete data received. Retrying (2/3)...
[2023-12-13 19:33:12,470: WARNING/ForkPoolWorker-1] WARNING: [youtube] Incomplete data received. Retrying (3/3)...
[2023-12-13 19:33:12,557: WARNING/ForkPoolWorker-1] WARNING: [youtube] Incomplete data received. Giving up after 3 retries
[2023-12-13 19:33:13,142: WARNING/ForkPoolWorker-1] vM6WtOmy190: upload comments
[2023-12-13 19:33:13,256: WARNING/ForkPoolWorker-1] 529ad129-17de-40be-bc8f-69ca227c66b9 success callback
[2023-12-13 19:33:13,258: INFO/ForkPoolWorker-1] Task download_pending[529ad129-17de-40be-bc8f-69ca227c66b9] succeeded in 19.881462368997745s: 'downloaded 1 videos.'
```

**Comment by bbilly1** [comment]
should be fixed in v0.4.5
